CHANGELOG.latest.md

```diff
@@ -1,5 +1,2 @@
-### New Features
-* [Paywalls] Add default locale (#1809) via Josh Holtz (@joshdholtz)
-* [Paywalls] Support to round+truncate prices displayed on paywalls for backend-specified countries (#1806) via James Borthwick (@jamesrb1)
-### Other Changes
-* Publish paywall tester on each release (#1803) via Toni Rico (@tonidero)
+### Dependency Updates
+* Update `compose-bom` to latest version (#1817) via Toni Rico (@tonidero)
```

Comment:
Might be a good idea to indicate the version here

Reply:
Also, maybe we should at least make it a minor?

Reply:
Yes to the version. As for making it a minor... Following semver, this isn't really a new feature, so it should be a patch? And I also don't think people would look that much closer a minor vs a patch IMO. Wdyt?

Reply:
We have determined the version number before based on dependencies updates, and since we are changing dependencies by a minor number, I thought this could be a minor. That was my reasoning.

Reply:
Ok that's fine yeah, will just regenerate the release PR then!